cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageHandler.java

```diff
@@ -109,18 +114,60 @@ public void getAccessesForRun(HttpRequest request, HttpResponder responder,
     responder.sendJson(HttpResponseStatus.OK, lineageStore.getRunMetadata(run), SET_METADATA_RECORD_TYPE, GSON);
   }
 
-  private void checkArguments(long start, long end, int levels) throws BadRequestException {
+  private void checkLevels(int levels) throws BadRequestException {
+    if (levels < 1) {
+      throw new BadRequestException(String.format("Invalid levels (%d), should be greater than 0.", levels));
+    }
+  }
+
+  // TODO: This is a fairly common operation useful in various handlers
```

Comment:
hmmm, sorry I may have missed this earlier, but then should this move to TimeMathParser itself?